graph/src/components/ethereum/adapter.rs

```diff
@@ -580,23 +578,21 @@ impl BlockStreamMetrics {
     pub fn new(
         registry: Arc<impl MetricsRegistry>,
         ethrpc_metrics: Arc<SubgraphEthRpcMetrics>,
-        deployment_id: SubgraphDeploymentId,
+        deployment_id: &SubgraphDeploymentId,
         stopwatch: StopwatchMetrics,
     ) -> Self {
         let blocks_behind = registry
-            .new_gauge(
-                format!("subgraph_blocks_behind_{}", deployment_id.to_string()),
-                String::from(
-                    "Track the number of blocks a subgraph deployment is behind the HEAD block",
-                ),
-                HashMap::new(),
+            .new_deployment_gauge(
+                "deployment_blocks_behind",
+                "Track the number of blocks a subgraph deployment is behind the HEAD block",
+                deployment_id.as_str(),
             )
             .expect("failed to create `subgraph_blocks_behind` gauge");
```

Comment:
Update error message to match the new metric name: `failed to create 'deployment_blocks_behind' gauge`.

Reply:
Done